specification/cdn/resource-manager/Microsoft.Cdn/stable/2019-12-31/cdn.json

```diff
@@ -3761,7 +3761,8 @@
             "ModifyRequestHeader",
             "ModifyResponseHeader",
             "UrlRedirect",
-            "UrlRewrite"
+            "UrlRewrite",
+            "DefaultOriginGroupOverride"
```

Comment:
This is the action name expose to customer. Have PMs agreed on this name?

Reply:
Yes. Thanks